package.json

```diff
@@ -142,6 +146,7 @@
     "test:unit": "cross-env NODE_ENV=test BABEL_ENV=test jest '(/test/unit.*\\.spec|/test/webui/.*\\.spec)\\.js' --maxWorkers 2",
     "test:func": "cross-env NODE_ENV=test BABEL_ENV=test jest '(/test/functional.*\\.func)\\.js' --maxWorkers 2",
     "pre:ci": "npm run lint && npm run build:webui",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
```

Comment:
What that means?

Reply:
This is called via commit hook, see https://github.com/marionebl/commitlint#getting-started or [commit-msg](https://git-scm.com/docs/githooks#_commit_msg) or [here](https://github.com/typicode/husky/blob/master/HOOKS.md) for the name matching between npm name and git.